src/main/java/bit/javaoop/AbstractEmployee.java

```diff
@@ -26,4 +26,6 @@ public double getSalary() {
     public void setSalary(double salary) {
         this.salary = salary;
     }
-}
+
+    abstract public String toStr(int line);
```

Comment:
Ok. Let's thing about it...

In Java we already have a `toString` method defined in the `Object` class which can be overridden and in general should be used to convert a class to it's string representation.

However this is not a standard `toString` method since it requires this additional `line` parameter. 
So now it's time to think whether it's this class responsibility to be able to print itself with a proper indentation and I think it's not. 

This class is pretty generic and can be used in multiple contexts. I would suggest to define a separate class like `CompanyPrinter` (you can think of a better name) which would take a `Company` and print all it's employees with a proper indentation :)